technologies/FlexBE/docs/hfsmbth.md

```diff
@@ -0,0 +1,39 @@
+# The Mythical HFSMBTH
```

Comment:
Might want to fully define this once as Hierarchical Finite-State Machine / Behavior Tree Hybrid.